extensions/binding/org.eclipse.smarthome.binding.yahooweather/src/main/java/org/eclipse/smarthome/binding/yahooweather/handler/YahooWeatherHandler.java

```diff
@@ -154,7 +156,9 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
     private synchronized boolean updateWeatherData() {
         try {
-            weatherData = getWeatherData();
+            weatherData = getWeatherData(
+                    "https://query.yahooapis.com/v1/public/yql?format=json&q=SELECT%20*%20FROM%20weather.forecast%20WHERE%20u=%27c%27%20AND%20woeid%20=%20%27"
```

Comment:
What is the difference between these two http urls that you now have put them into the method argument instead of treating it as an internal information of the method, which url needs to be called?